src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Endpoints.cs

```diff
@@ -704,6 +704,37 @@ internal partial class OpenIdConnectServerHandler : AuthenticationHandler<OpenId
             }
         }
 
+        private bool TryGetClientCredentialsFromHeader(OpenIdConnectMessage request)
```

Comment:
Looks like this method is only used by `InvokeTokenEndpointAsync`. Consider reverting this change.